README.md

```diff
@@ -30,15 +33,45 @@ to ensure that compliance is granted only to the exact binary tested.
 FIPS compliance was [granted](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4407)
 to an old version of BoringSSL that was tested with `Clang 12.0.0`.
 
-Installing `Clang 12.0.0` on modern environments is at best difficult. For `linux` `x86_64`, we work around
-this problem by shipping the pre-built binaries under `vendor/boringssl-fips/linux_x86_64`. We then tell the BoringSSL libraries
-to use this path by setting the `BORING_BSSL_PATH` environment variable in `.cargo/config.toml`.
+Given that FIPS support will always have special environmental build requirements, we currently we work around this by vendoring OS/arch specific FIPS-compliant binary builds of `boringssl` in [](vendor/boringssl-fips/)
+
+We vendor FIPS boringssl binaries for 
+- `linux/x86_64`
+- `linux/arm64`
+
+To use these vendored libraries and build ztunnel for either of these OS/arch combos, for the moment you must manually edit
+[.cargo/config.toml](.cargo/config.toml) and change the values of BORING_BSSL_PATH and BORING_BSSL_INCLUDE_PATH under the `[env]` key to match the path to the vendored libraries for your platform, e.g:
+
+#### For linux/x86_64
+``` toml
+BORING_BSSL_PATH = { value = "vendor/boringssl-fips/linux_x86_64", force = true, relative = true }
```

Comment:
Do you think its feasible to get the boring crate to allow some form of substition so we can just put `vendor/boringssl-fips/{platform}` or whatever?

Reply:
Possibly/maybe - I think once the PRs @PiotrSikora and @nmittler already have outstanding in that repo get merged, that's something I'll look at if no one beats me to it.